tests/tt_metal/tt_metal/test_kernels/dataflow/unit_tests/command_queue/spoof_prefetch_host.cpp

```diff
@@ -0,0 +1,59 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
think this should use the existing spoof_prefetch and use an ifdef for main (or some such).  that way if we change the handshake to dispatcher we have fewer places to fix.

actually, I'm in the midst of a refactor to put those shared components into a header file to be shared across prefetch/dispatch variations (host/dram, etc).  won't get it done today, though...

Reply:
btw on `abhullar/fd2-host` I have moved some of the consts to `command_queue_interface.hpp`